Mathlib/AlgebraicGeometry/EllipticCurve/Projective/Basic.lean

```diff
@@ -222,9 +221,8 @@ variable (W') in
 /-- The polynomial `W(X, Y, Z) := Y²Z + a₁XYZ + a₃YZ² - (X³ + a₂X²Z + a₄XZ² + a₆Z³)` associated to a
 Weierstrass curve `W` over a ring `R` in projective coordinates.
 
-This is represented as a term of type `MvPolynomial (Fin 3) R`, where `X`, `Y`, and `Z`
-represent `X`, `Y`, and `Z` respectively. -/
-noncomputable def polynomial : MvPolynomial (Fin 3) R :=
+This is represented as a term of type `R[X,Y,Z]`, which is secretly `MvPolynomial (Fin 3) R`. -/
```

Comment:
Personally I think this actually *reduces* readability, because now there is a secret when before there wasn't.

Reply:
I disagree here: one could argue that notation is always creating a secret, but notation is often good. Here, `R[X,Y,Z]` is clearly a completely standard math notation, that we should support if we can.

Reply:
Indeed, and given that we *already* have (scoped) notation `R[X]` for `Polynomial R`, I don't think there is any problem with this.